source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -120,14 +120,24 @@ def _getOffsetFromPoint(self,x,y):
 		else:
 			raise NotImplementedError
 
-	def _getPointFromOffset(self,offset):
+	@classmethod
+	def _getPointFromOffsetInObject(cls,obj,offset):
 		try:
-			res=self.obj.IAccessibleTextObject.characterExtents(offset,IAccessibleHandler.IA2_COORDTYPE_SCREEN_RELATIVE)
+			res=obj.IAccessibleTextObject.characterExtents(offset,IAccessibleHandler.IA2_COORDTYPE_SCREEN_RELATIVE)
 		except COMError:
 			raise NotImplementedError
+		# Normally, L{_getPointFromOffset} implementations that rely on
+		# character bounding rectangles use the center point of the rectangle.

```

Comment:
Is this comment still correct? It makes it sound like this will not use the center, which is what it appears to do.